internal/pkg/source/testdata/src/domination/test.go

```diff
@@ -25,7 +25,7 @@ type foo struct {
 }
 
 func f1() {
-	f := &foo{name: "n", password: "p"}
+	f := &foo{name: "n", password: "p"} // want `sanitizer\(t0\) sink\(t0\) `
```

Comment:
Comment: I can't find it in the SSA documentation, so I might be misremembering, but I think register names like `t0` are in that *deterministic but undefined* category.  I still consider this to be an improvement in the tests, just wanted to mention it.

Reply:
I agree - this is a bit of an unknown - we may need to revisit this.

Where possible, I tried to avoid relying on `t#`